bazarr/subtitles/indexer/series.py

```diff
@@ -41,13 +41,13 @@ def store_subtitles(original_path, reversed_path, use_cache=True):
                                                               use_cache=use_cache)
                     for subtitle_language, subtitle_forced, subtitle_hi, subtitle_codec in subtitle_languages:
                         try:
-                            if (settings.general.ignore_pgs_subs and subtitle_codec.lower() == "pgs") or \
-                                    (settings.general.ignore_vobsub_subs and subtitle_codec.lower() ==
-                                     "vobsub") or \
-                                    (settings.general.ignore_ass_subs and subtitle_codec.lower() ==
-                                     "ass"):
-                                logging.debug("BAZARR skipping %s sub for language: %s" % (subtitle_codec, alpha2_from_alpha3(subtitle_language)))
-                                continue
+                            skip_flag = False
+                            for sub_codec in settings.general.ignored_subs:
+                                if subtitle_codec.lower() == sub_codec:
+                                    skip_flag = True
+                                    logging.debug("BAZARR skipping %s sub for language: %s" % (subtitle_codec, alpha2_from_alpha3(subtitle_language)))
+                                    break
+                            if skip_flag: continue
```

Comment:
I would say we could use some refactoring here to completely skip this boolean like:

```
if any(subtitle_codec.lower() == sub_codec for sub_codec in settings.general.ignored_subs):
        logging.debug(f"BAZARR skipping {subtitle_codec} sub for language: {alpha2_from_alpha3(subtitle_language)}")
        continue
```

Similar comment also applies to the movies indexer.

Reply:
I think I did the rebase/merge/refactor appropriately? I'm sorry, I'm still quite new to this and I managed to break my dev environment so I couldn't do a proper test before heading out of town shortly here.